site/app/controllers/HomePageController.php

```diff
@@ -290,10 +290,40 @@ public function createCoursePage() {
                 'showCourseCreationPage',
                 $faculty ?? null,
                 $this->core->getUser()->getId(),
-                $this->core->getQueries()->getAllUnarchivedSemester(),
+                //$this->core->getQueries()->getAllUnarchivedSemester(),
```

Comment:
is there a reason why you are keeping this commented out? You probably just want to remove it

Reply:
yeah, will remove